pkgs/by-name/md/mdns-reflector/package.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, unstableGitUpdater, cmake, lib }: stdenv.mkDerivation rec {
```

```suggestion
{ stdenv
, fetchFromGitHub
, unstableGitUpdater
, cmake
, lib
}: stdenv.mkDerivation rec {
```

common practice is to break package arguments into seperate lines when there are more than 3